### PR TITLE
Update NoiseSuppressionDefault type in xml-config-file.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8974.

### CODE REVIEW

File: Teams/rooms/xml-config-file.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ If a variable value is of the wrong type, elements are out of order, elements ar
...
-| `<NoiseSuppressionDefault>`                 | Boolean &#x2777;            | First &#x2776; | Controls noise suppression levels in Teams.<br><ul><li><b>0</b> Off. Use OEM-provided noise suppression only.</li><li><b>1</b> Auto (default). Teams decides on the best level of noise suppression based on local noise.</li><li><b>2</b> Low. Suppresses low levels of persistent background noise, such as a computer fan or air conditioner.</li><li><b>3</b> High. Suppresses all background sound that isn't speech.</li></ul>
+| `<NoiseSuppressionDefault>`                 | Number;            | First &#x2776; | Controls noise suppression levels in Teams.<br><ul><li><b>0</b> Off. Use OEM-provided noise suppression only.</li><li><b>1</b> Auto (default). Teams decides on the best level of noise suppression based on local noise.</li><li><b>2</b> Low. Suppresses low levels of persistent background noise, such as a computer fan or air conditioner.</li><li><b>3</b> High. Suppresses all background sound that isn't speech.</li></ul>
```
Comment (JohanFreelancer9) [inline Teams/rooms/xml-config-file.md:138]:
```suggestion
| `<NoiseSuppressionDefault>`                 | Number            | First &#x2776; | Controls noise suppression levels in Teams.<br><ul><li><b>0</b> Off. Use OEM-provided noise suppression only.</li><li><b>1</b> Auto (default). Teams decides on the best level of noise suppression based on local noise.</li><li><b>2</b> Low. Suppresses low levels of persistent background noise, such as a computer fan or air conditioner.</li><li><b>3</b> High. Suppresses all background sound that isn't speech.</li></ul>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
Fixed in another PR.

**Pull request closed without merging**